src/Lit.fs

```diff
@@ -12,6 +13,13 @@ module Bindings =
     let render (t: TemplateResult) (el: HTMLElement): unit = importMember "lit-html"
     let styleMap (styles: obj): obj = importMember "lit-html/directives/style-map"
 
+    let classMap (classes: obj): obj = importMember "lit-html/directives/class-map"
+    let private _until: obj = import "until" "lit-html/directives/until"
+
+    [<Emit("Bindings__until(...$0)")>]
```

Comment:
@alfonsogarciacaro HELP! I didn't know how to make it work so I had to make an intermediate `_until` variable to contain that, the method requires the parameters to be spread rather than taking a simple array I'm not sure how to make that work

Reply:
We can use `[<ParamArray>]` attribute but we need to convert the bindings into a class or an interface, I can do that :+1: